src/components/Buttons/Button/Button.tsx

```diff
@@ -39,17 +39,25 @@ export const Button = forwardRef<HTMLButtonElement | HTMLAnchorElement, Props>(
             children,
             disabled = false,
             isBlock = false,
-            isInline = false,
             isLoading = false,
-            isLink = false,
-            isPrimary = false,
             type = 'button',
             href,
             size = 'normal',
+            variant = 'filled',
+            context = 'primary',
```

Comment:
I'm not sure these are good default. It will change the default button from a neutral to a primary look. this will make migration more difficult. But if this is the main button we are going to use from here on, it might make sense. Can you check with Dima?

I also think the BREAKING CHANGE description of the PR is not explicit about it, we should add it once we are sure.